packages/shared/src/components/Icon.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames';
+
+export type Size = 'small' | 'medium' | 'large';
```

Comment:
Since this is now an exported type, I think it would be better to be more explicit about it from `Size` to `IconSize`.
```suggestion
export type IconSize = 'small' | 'medium' | 'large';
```